public/app/features/dashboard-scene/utils/interactions.ts

```diff
@@ -136,6 +136,17 @@ export const DashboardInteractions = {
 
 const reportDashboardInteraction: typeof reportInteraction = (name, properties) => {
   const meta = isScenesContextSet ? { scenesView: true } : {};
+  const isDynamicDashboard = config.featureToggles?.dashboardNewLayouts ?? false;
```

Comment:
should we call is DynamicDashboards or newLayout ?

Reply:
I think calling it dynamic dashboards is fine.